coderd/workspaces.go

```diff
@@ -676,7 +676,7 @@ func createWorkspace(
 		if req.TemplateVersionID != uuid.Nil {
 			builder = builder.VersionID(req.TemplateVersionID)
 		}
-		if req.EnableDynamicParameters {
+		if req.EnableDynamicParameters && api.Experiments.Enabled(codersdk.ExperimentDynamicParameters) {
```

Comment:
@aslilac thoughts on this?

If it is good, we can put this in. And add into the experiment notes wherever that validation is super lax